warehouse/client/controlplane/client.go

```diff
@@ -25,6 +25,7 @@ type BasicAuth struct {
 
 type InternalControlPlane interface {
```

Comment:
[minor] should this interface be defined only on the validations package? Are there other packages that specifically depend on `GetDestinationSSHKeys` and `GetSSHKeys`?